Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM python:3.6.12-buster AS compiler
```

Comment:
update python version to FROM python:3.8-buster to be consistent with other Dockerfiles